cli/lib/kontena/cli/cloud/master/add_command.rb

```diff
@@ -1,6 +1,3 @@
-require 'kontena/cli/cloud_command'
-require 'kontena/cli/cloud/master_command'
-
 module Kontena::Cli::Cloud::Master
```

Comment:
I suppose those requires were there to make sure there's `Kontena::Cli::Cloud::Master` module tree. It would usually be better to just nest them like `module Kontena; module Cli; module ........`.